net-misc/cadaver/cadaver-0.23.3.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="A command line WebDAV client"
+DESCRIPTION="It supports file upload, download, on-screen display, namespace \
+operations (move/copy), collection creation and deletion, and locking \
+operations."
+HOMEPAGE="http://www.webdav.org/cadaver/"
+COPYRIGHT="1998-2008 Joe Orton
+	2002-2003 GRASE Lab, UCSC"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://www.webdav.org/cadaver/cadaver-$portVersion.tar.gz"
+CHECKSUM_SHA256="fd4ce68a3230ba459a92bcb747fc6afa91e46d803c1d5ffe964b661793c13fca"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	cadaver = $portVersion
+	cmd:cadaver
+	"
+REQUIRES="
+	haiku
+	lib:libexpat
+	lib:libiconv
+	lib:libintl
+	lib:libncurses
+	lib:libreadline
+	lib:libssl
+	lib:libxml2
+	lib:libzlib
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libexpat
+	devel:libiconv
+	devel:libintl
+	devel:libncurses
+	devel:libreadline
+	devel:libssl
+	devel:libxml2
+	devel:libzlib
```

Comment:
```diff
-	devel:libzlib
+	devel:libz
```

Reply:
@pjht be sure to check the build with haikuporter, this would have failed from the start :)

Reply:
It works fine

Reply:
Yeah, I've looked it up and seems that a few recipes also use this "devel:libzlib", not sure where it is coming from though, zlib doesn't provide it, @fbrosson any idea there (there are 4 recipes using that one as well) ...